make/catalog.mk

```diff
@@ -10,6 +10,8 @@ OPM_DOCKERFILE_TAG ?= latest
 $(CATALOG_DOCKERFILE): $(OPM)
 	-mkdir -p $(PROJECT_DIR)/catalog/authorino-operator-catalog
 	cd $(PROJECT_DIR)/catalog && $(OPM) generate dockerfile authorino-operator-catalog -b "quay.io/operator-framework/opm:${OPM_DOCKERFILE_TAG}" -i "quay.io/operator-framework/opm:${OPM_DOCKERFILE_TAG}"
+	# Inject --pprof-addr="" into both RUN and CMD serve invocations to disable running the pprof server
+	sed -i.bak -E '/(opm".*"serve|^CMD \["serve)/s#\]$$#, "--pprof-addr="]#' $(CATALOG_DOCKERFILE) && rm -f $(CATALOG_DOCKERFILE).bak
```

Comment:
According to:
- https://github.com/operator-framework/operator-registry/blob/a5f1128789836e8e1d1569b6f3fb47b8194deb86/cmd/opm/serve/serve.go#L212-L220

Reply:
So we add the `--pprof-addr` flag, _set to an empty string value_, and this will make profiler interface's `isEnabled()` function return true?

Reply:
Or is it the other way around? We want it to be empty so `isEnabled() == false`, to short circuit starting the endpoint?

Reply:
Yes, we want it to be `false` to short circuit the endpoint

Reply:
Although, I suppose alternatively we can set a random port using `-p` if we want the endpoint running 🤔